compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -3043,14 +3044,22 @@ impl<'ast, 'ra: 'ast, 'tcx> LateResolutionVisitor<'_, 'ast, 'ra, 'tcx> {
             "missing lifetime specifier{}",
             pluralize!(num_lifetimes)
         );
-        self.add_missing_lifetime_specifiers_label(
-            &mut err,
-            lifetime_refs,
-            function_param_lifetimes,
-        );
+        if in_precise_capture {
+            err.note(
+                "this function's return type specifies a borrowed `use<'_>` with an elided \
```

Comment:
"a borrowed `use<'_>`" feels really awkward. I think this can be a bit more direct. Perhaps something like:

"cannot capture elided lifetime with `use<'_>` when there is no lifetime in scope to capture"